ACadSharp/Entities/Dimension.cs

```diff
@@ -106,18 +107,36 @@ public DimensionType Flags
 		public double Measurement { get; internal set; }
 
 		/// <summary>
-		/// Dimension text explicitly entered by the user
+		/// Gets the dimension text explicitly entered by the user or the formatted
+		/// measurement value or sets the dimension text.
 		/// </summary>
-		/// <remarks>
-		/// Optional; default is the measurement.
-		/// If null, the dimension measurement is drawn as the text, 
-		/// if ““ (one blank space), the text is suppressed.Anything else is drawn as the text
-		/// </remarks>
+		/// <value>
+		/// If null or empty, the formatted <see cref="Measurement"/> value is returned, 
+		/// if ““ (one blank space), the text is suppressed. Anything else is drawn as the text.
+		/// </value>
 		[DxfCodeValue(DxfReferenceType.Optional, 1)]
 		public string Text
 		{
-			get { return string.IsNullOrEmpty(_text) ? this.Measurement.ToString() : this._text; }
-			set { this._text = value; }
+			get {
+				if (this._text == " ")
```

Comment:
I would put this if after the `IsNullOrEmpty` and then check the _text using the method `string.IsNullOrWhiteSpace` due this does not check for a string with multiple white spaces.

Reply:
... Anything else is drawn as the text. This includes multiple white spaces. Hmm - not sensible.

anyhow - after some work with dimensions I think my proposal is not a good idea, since the generated text may be incomplete or not suitable for angular dimensions. And there is no indication, whether the text was created according to the various properties or explicitly entered. Please see #223.